zerver/tests/test_docs.py

```diff
@@ -128,9 +128,10 @@ def test_api_doc_endpoints(self) -> None:
         ENDPOINT_REGEXP = re.compile(r"href=\"/api/\s*(.*?)\"")
         endpoint_list_set = set(re.findall(ENDPOINT_REGEXP, api_page_raw))
         endpoint_list = [f"/api/{endpoint}" for endpoint in endpoint_list_set]
+
         # Validate that the parsing logic isn't broken, since if it
         # broke, the below would become a noop.
-        self.assertGreater(len(endpoint_list), 70)
+        self.assert_length(endpoint_list, 110)
```

Comment:
It's useful when looking at a test to know exactly how expensive it is.

Reply:
I'll just make the number more realistic (but imprecise), so that future people like me that profile the test get a quicker idea of how many endpoints there are.

Reply:
If you simply assert that the list is non-zero, you still have the danger that some change to the regex or url structure or way of putting the links into the main document silently removes coverage.

Reply:
Well, you can change the number to asserting it's at least 110. I just can guarantee it'll be annoying to update every time we add a new documented API endpoint, and I think that annoyingness outweighs any downside for this number being slightly imprecise.